point_cloud_transport/include/point_cloud_transport/publisher_plugin.hpp

```diff
@@ -92,7 +94,7 @@ class PublisherPlugin
 
   //! Publish a point cloud using the transport associated with this PublisherPlugin.
   POINT_CLOUD_TRANSPORT_PUBLIC
-  virtual void publish(const sensor_msgs::msg::PointCloud2::ConstSharedPtr & message) const;
+  virtual void publishPtr(const sensor_msgs::msg::PointCloud2::ConstSharedPtr & message) const;
```

Comment:
Out of curiosity, why rename this?

Reply:
Two main reasons:
 - I was getting this warning
```bash
hidden overloaded virtual function 'point_cloud_transport::PublisherPlugin::publish' declared here: different number of parameters (1 vs 2)
```
 - To have the same API as image_Transport